app/helpers/link_helper.rb

```diff
@@ -0,0 +1,23 @@
+module LinkHelper
+  EXTERNAL_LINK_CLASS = 'usa-link--external'.freeze
+
+  def new_window_link_to(name = nil, url = nil, html_options = nil, &block)
+    html_options ||= {}
+    html_options[:class] ||= html_options[:class].to_s
+    html_options[:target] = '_blank'
+
+    classes = html_options[:class].split(' ').append(EXTERNAL_LINK_CLASS)
+
+    html_options[:class] = classes.uniq.join(' ')
+
+    link_to(url, html_options) do
+      if block_given?
+        yield(block)
+        concat content_tag('span', t('links.new_window'), class: 'usa-sr-only')
```

Comment:
How is the block form of this method expected to be called? With `link_to`, the arguments are shifted so make `name` optional:

https://github.com/rails/rails/blob/60ef07/actionview/lib/action_view/helpers/url_helper.rb#L197

Do we even need the block support? It doesn't appear we're using it?

Aside: Reminds me that test specs would be good as well.

Reply:
We do use the block version in the footer here: https://github.com/18F/identity-idp/pull/4310/files#diff-727c1af48666bc6f26041f22d5bca5b0f0ab95014c24d020434389879134191cR32-R36